test/connectors/parsers/OtherEmploymentIncomeHttpParserSpec.scala

```diff
@@ -20,60 +20,66 @@ import connectors.errors.{ApiError, SingleErrorBody}
 import play.api.http.Status._
 import play.api.libs.json.{JsValue, Json}
 import support.UnitTest
-import support.builders.api.OtherEmploymentsIncomeBuilder.{anOtherEmploymentsIncome, anOtherEmploymentsIncomeJson}
+import support.builders.api.OtherEmploymentIncomeBuilder.{anOtherEmploymentIncome, anOtherEmploymentsJson}
 import uk.gov.hmrc.http.HttpResponse
 
-class OtherEmploymentsIncomeHttpParserSpec extends UnitTest {
+class OtherEmploymentIncomeHttpParserSpec extends UnitTest {
 
   private val anyHeaders: Map[String, Seq[String]] = Map.empty
   private val anyMethod: String = "GET"
   private val anyUrl = "/any-url"
   private val singleErrorBody: SingleErrorBody = SingleErrorBody("PARSING_ERROR", "Error parsing response from DES")
   private val singleErrorBodyJson: JsValue = Json.toJson(singleErrorBody)
-  private val underTest = OtherEmploymentsIncomeHttpParser.OtherEmploymentsIncomeHttpReads
+  private val employmentOtherIncomeHttpReads = OtherEmploymentIncomeHttpParser.OtherEmploymentIncomeHttpReads
```

Comment:
We use to have the `underTest` pattern everywhere, I'd keep it like the same way for now if possible? We could have a group discussion about it

Reply:
Sure. I'll keep it the same for consistency and then we can refactor it later for all as a separate Tech Debt.